blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/preload/OrganizationStatePreloader.java

```diff
@@ -44,4 +66,26 @@ public String getStateJson() {
             return "{}";// if will happen only when there is no implementation of BlueOrganization found.
         }
     }
+
+    private static final Pattern pattern = Pattern.compile("\\/organizations\\/([^\\/]*)\\/");
```

Comment:
this pattern looks wrong - the `/` is not special (unless I missed something) so does not need escaping with `\`?